c-dependencies/js-compute-runtime/Makefile

```diff
@@ -59,7 +59,10 @@ $(RUST_URL_LIB): $(RUST_URL_RS_FILES) $(RUST_URL_SRC)/Cargo.toml $(RUST_URL_SRC)
 %.o: $(FSM_SRC)%.cpp $(FSM_SRC)Makefile $(RUST_URL_LIB) compiler_flags
 	$(WASI_CXX) $(CXX_FLAGS) $(CXX_OPT) $(DEFINES) -I $(SM_SRC)include -MMD -MP -c -o $@ $<
 
-builtins/%.o: $(FSM_SRC)builtins/%.cpp $(FSM_SRC)Makefile $(RUST_URL_LIB) compiler_flags
+builtins:
+	$Q mkdir -p builtins
+
+builtins/%.o: $(FSM_SRC)builtins/%.cpp $(FSM_SRC)Makefile $(RUST_URL_LIB) compiler_flags | builtins
```

Comment:
This fixes local builds, as when building in ci we pull down the wasm from a previous step and use that instead.

Reply:
Is it desirable to build in `$OUT_DIR` when building locally? Or is this a sign that `build.rs` should change? I don't have an opinion either way, I just wonder about things sometimes.

Reply:
That's a point, we could use the presence of `$OUT_DIR` to detect when we're building from build.rs, or perhaps have a default that sets it to the path of the makefile when it's missing. That seems like a reasonable change, but probably we should just take the time to switch this over to meson as you mentioned :)